hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALPrettyPrinter.java

```diff
@@ -344,9 +362,17 @@ public void processFile(final Configuration conf, final Path p)
     }
   }
 
-  public static void printCell(PrintStream out, Map<String, Object> op, boolean outputValues) {
-    out.println("row=" + op.get("row") + ", type=" + op.get("type") + ", column=" +
-      op.get("family") + ":" + op.get("qualifier"));
+  public static void printCell(PrintStream out, Map<String, Object> op,
+    boolean outputOnlyRowKey, boolean outputValues) {
```

Comment:
nit: when changing existing method signatures to add more parameters, a good practice is to add the new parameters at the end of parameters list. That helps when fixing compile errors on possible existing clients.